lib/nitcorn/file_server.nit

```diff
@@ -106,7 +106,7 @@ class FileServer
 					end
 				end
 
-				var is_dir = local_file.file_stat.is_dir
+				var is_dir = local_file.file_stat.as(not null).is_dir
```

Comment:
if you use a `file_exists`, then a `file_stat.as(not null)`  then just use `file_stat`

~~~nit
vat stat = local_file.file_stat
if stat != null then # was if local_file.file_exists then
  if stat.is_dir then #  was if local_file.file_stat.is_dir then
  # ...
~~~